siad/daemon.go

```diff
@@ -24,6 +28,20 @@ func startDaemon() error {
 	// Establish multithreading.
 	runtime.GOMAXPROCS(runtime.NumCPU())
 
+	// Print a startup message.
+	fmt.Println("siad is loading")
```

Comment:
I don't think startup messages are necessary. Maybe leave the `"loaded in %d seconds"`, but inside a `if config.Siad.Profile` block.